04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,74 @@
+const Product = require('../models/product')
+
+const getAllProductsStatic = async(req, res)=>{
+   
+    const products = await Product.find({}).select('name price').limit(10)
+    res.status(200).json({products, nbHits: products.length})
+}
+
+const getAllProducts = async(req, res)=>{
+    const {featured, company, name, sort, fields, numericFilters} = req.query;
+    const queryObject = {}
+
+    if(featured){
+        queryObject.featured = featured === 'true'? true: false
```

Comment:
the statement `featured === 'true' returns a boolean in and of itself so that would be the same as what you have here with the ternary operator. 

We don't always have control of these things but this is a good usecase of an optional param where you could say "was featured passed at all?" instead of checking its value against the string 'true'